src/CommunityToolkit.Maui.UnitTests/Converters/ColorToColorConverter_Tests.cs

```diff
@@ -5,75 +5,75 @@ namespace CommunityToolkit.Maui.UnitTests.Converters;
 
 public class ColorToColorConverter_Tests : BaseTest
 {
-	public static IReadOnlyList<object?[]> ColorToBlackOrWhiteData { get; } = new[]
```

Comment:
Please check if it works. According to xunit documentation it should be object

Reply:
<img width="1092" alt="Screen Shot 2022-03-21 at 4 35 07 PM" src="https://user-images.githubusercontent.com/13558917/159380078-0c2c3b24-57e7-436f-89c2-351ebe4b5e60.png">
<img width="1792" alt="Screen Shot 2022-03-21 at 4 36 04 PM" src="https://user-images.githubusercontent.com/13558917/159380083-458f9b88-4600-41f4-b165-d01c536e8af0.png">
It does! I was surprised too